x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -223,6 +223,8 @@ static RoleDescriptor kibanaSystem(String name) {
                     .privileges("all")
                     .allowRestrictedIndices(true)
                     .build(),
+                // Fleet writes to this datastream for Agent status alerting feature
+                RoleDescriptor.IndicesPrivileges.builder().indices("logs-elastic_agent.status_change-*").privileges("all").build(),
```

Comment:
__question:__

1. Is the functionality that relies on this permission available in all our offerings (on-prem, ECH, Serverless)?
2. Are you planning to backport this change to 8.x?
3. The linked issue says that `user needs permissions to **write**`, do we really need `all` here or we can just use `write`?